configure.ac

```diff
@@ -3527,7 +3527,9 @@ AC_CHECK_FUNCS(alarm accept4 setitimer getitimer bind_textdomain_codeset chown \
  getgrouplist getgroups getlogin getloadavg getpeername getpgid getpid \
  getpriority getresuid getresgid getpwent getpwnam_r getpwuid_r getspnam getspent getsid getwd \
  if_nameindex \
- madvise mkfifoat mknod mknodat mktime mremap nice openat pathconf pause pipe2 plock poll \
```

Comment:
What and why did you modify the configure script?

Reply:
In fact I got the fixes from a HP-UX specialist, if it works without the better.

Reply:
will remove the changes ...

Reply:
The change comes actually from the original PR which broke it, https://github.com/python/cpython/commit/51aa35e9e17eef60d04add9619fe2a7eb938358c#diff-67e997bcfdac55191033d57a16d1408a too much has been removed as opposed to the `configure` script.

Reply:
The `configure.ac` needs to retain:

```
initgroups kill killpg lchmod lchown lockf linkat lstat lutimes mmap \
memrchr mbrtowc mkdirat mkfifo \
```

Reply:
Here is the patch I have sent via email:

```
# diff -U 1 configure.ac.orig configure.ac
--- configure.ac.orig   2019-05-28 14:34:58 +0000
+++ configure.ac        2019-05-28 14:38:23 +0000
@@ -3526,6 +3526,8 @@
 getgrouplist getgroups getlogin getloadavg getpeername getpgid getpid \
 getpriority getresuid getresgid getpwent getpwnam_r getpwuid_r getspnam getspent getsid getwd \
 if_nameindex \
+ initgroups kill killpg lchmod lchown lockf linkat lstat lutimes mmap \
+ memrchr mbrtowc mkdirat mkfifo \
 mkfifoat mknod mknodat mktime mremap nice openat pathconf pause pipe2 plock poll \
 posix_fallocate posix_fadvise posix_spawn posix_spawnp pread preadv preadv2 \
 pthread_condattr_setclock pthread_init pthread_kill putenv pwrite pwritev pwritev2 \
```

Please reinstantiate otherwise all of these won't be available at compile time.

Reply:
I'm sorry about I merged the previous PR.  I thought travis checked configure script is really generated from configure.ac.